README.md

```diff
@@ -13,16 +13,16 @@ TODO: Description
 You can install the latest release via [`pip`](https://pypi.org/project/pip/):
 
 ```bash
-pip install <PYPI_NAME>
+pip install docs-build
```

Comment:
do we plan on publishing this to pypi?
also, i think the name is too generic as `docs-build`.. it should maybe called `ape-docs-build` or something.

Reply:
For now then, we should delete this part of the README, because `pip install ape-docs-build` won't work without that.

That being said, we should definitely include instructions for how to use this in your Ape plugin!